src/OpenTracing/Mock/Propagators.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Collections.Generic;
+using OpenTracing.Propagation;
+
+namespace OpenTracing.Mock
+{
+    public static class Propagators
+    {
+        public static readonly IPropagator Printer = new PrinterPropagator();
+
+        public static readonly IPropagator TextMap = new TextMapPropagator();
+    }
+
+    /// <summary>
+    /// Allows the developer to inject into the <see cref="MockTracer.Inject"/> and <see cref="MockTracer.Extract"/> calls.
+    /// </summary>
+    public interface IPropagator
+    {
+        void Inject<TCarrier>(MockSpanContext context, IFormat<TCarrier> format, TCarrier carrier);
+
+        MockSpanContext Extract<TCarrier>(IFormat<TCarrier> format, TCarrier carrier);
+    }
+
+    public sealed class PrinterPropagator : IPropagator
```

Comment:
I think ConsolePropagator would be clearer than Printer.

Reply:
It has the current name because it was ported from Java but I definitely agree that `ConsolePropagator` would be a better name (and more common in .NET). 

I think this would be a reasonable deviation as it's not in the core API and as it makes it much easier to understand.

Anyone has an issue with `PrinterPropagator` being renamed to `ConsolePropagator`?

Reply:
I think that's actually a good deviation, so +1 on that renaming.

Reply:
I changed the name to `ConsolePropagator`